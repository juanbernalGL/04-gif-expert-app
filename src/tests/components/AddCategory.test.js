import { waitForElementToBeRemoved } from '@testing-library/react';
import React from 'react';
const { shallow } = require("enzyme");
const { AddCategory } = require("../../components/AddCategory");

describe('test on <AddCategory></AddCategory>', () => {
    const setCategories = () => {};
    const wrapper = shallow(<AddCategory setCategories={setCategories} />);
    test('must show correctly', () => {
        expect( wrapper ).toMatchSnapshot();
    });

    test('must be change text input', () => {
        const input = wrapper.find('input');
        const value = 'Samurai X'
        input.simulate('change', { 
            target: {
                value
            }
        });
        
        expect( wrapper.find('p').text().trim() ).toBe(value) ;
    });

    test('should not post the information ', () => {
        
    });
});