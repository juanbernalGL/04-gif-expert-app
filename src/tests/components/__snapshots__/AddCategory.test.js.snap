// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on <AddCategory></AddCategory> must show correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <p />
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
